### ISSUE TITLE
Building from source fails on Mac

### ISSUE THREAD

**Issue by jgrant12345** [MovingBlocks/Terasology#5001]
### General Info

Terasology Launcher Version: 
Terasology Version: 
Operating System: Mac OS 
Onboard / Dedicated Graphics: M1 chip
Java Version: openjdk version "11.0.12" 2021-07-20

### What you were trying to do
I was trying to start the game from source and the game crashes whenever I do. First I tried using the configuration TerasologyPC then TerasologyPC for mac. They both give me the same error as is shown in the following screenshot
<img width="1314" alt="Screen Shot 2022-03-11 at 8 45 41 PM" src="https://user-images.githubusercontent.com/44660375/158004072-f39a29b3-af8d-414b-9786-43c6fb51c4b2.png">

Then I tried to run the commands in the command line ```gradlew game``` and ```gradlew jar game```. Both of which yield the following 
<img width="1278" alt="Screen Shot 2022-03-11 at 8 49 22 PM" src="https://user-images.githubusercontent.com/44660375/158004158-e0b4c250-6de0-4eee-b349-36aec466f90f.png">

A clear and concise description of what didn't match your expectations, a.k.a. the crash or malfunction you faced.

### How to reproduce

1. clone the repository
2. try to run with any of the commands such as gradlew game or gradlew jar game or run with the configuration in intellij (either 
TerasologyPC for Mac or TerasologyPC).

**Comment by jdrueckert** [comment]
Hi @jgrant12345,
thank you for filing this issue.
From the top of my head, I'm assuming either an issue with the M1 architecture or with your JDK.

I'd like to ask one of our Mac people @DarkWeird @Cervator @skaldarnar to try to reproduce this issue.

**Comment by DarkWeird** [comment]
> Hi @jgrant12345,
> thank you for filing this issue.
> From the top of my head, I'm assuming either an issue with the M1 architecture or with your JDK.
> 
> I'd like to ask one of our Mac people @DarkWeird @Cervator @skaldarnar to try to reproduce this issue.

I haven't mac now. (My company's mac. Not M1)

I see two possible source of this issue(possible outdated):
1. Splash screen is enabled (seems switch for it was broken picocli integration.
2.unsupported architecture (arm64 with mac)

Fixes for:
1 - my pr for separate graphic thread, or truly disable splash screen
2 - update lwjgl and add mac-arm64 natives (or nothing.. iirc new macs have translation layer for running x64 apps over arm64(like WoW system in Windows)

**Comment by jdrueckert** [comment]
@DarkWeird Thanks for your input :pray:  I hope that @Cervator or @skaldarnar can reproduce this, so we can investigate this better. If the root cause turns out to be the splash screen or architecture, we can definitely revisit your fix proposals :+1:

**Comment by skaldarnar** [comment]
When trying to build Terasology on an M1 Mac I see the following error:

```
> ./gradlew game

...

* What went wrong:
Execution failed for task ':engine:generateProto'.
> Could not resolve all files for configuration ':engine:protobufToolsLocator_protoc'.
   > Could not find protoc-3.17.0-osx-aarch_64.exe (com.google.protobuf:protoc:3.17.0).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.17.0/protoc-3.17.0-osx-aarch_64.exe
```

Building and running the launcher on M1 Macs seems to work fine.

Trying to start a downloaded game (v5.3.0-rc.2) did not work, though. The errors look like the come from missing libraries, likely because we don't ship the M1 natives in any form.

```
16:02:26.294 | INFO  | Game output: [LWJGL] Failed to load a library. Possible solutions:
16:02:26.295 | INFO  | Game output: 	a) Add the directory that contains the shared library to -Djava.library.path or -Dorg.lwjgl.librarypath.
16:02:26.295 | INFO  | Game output: 	b) Add the JAR that contains the shared library to the classpath.
16:02:26.295 | INFO  | Game output: [LWJGL] Enable debug mode with -Dorg.lwjgl.util.Debug=true for better diagnostics.
16:02:26.295 | INFO  | Game output: [LWJGL] Enable the SharedLibraryLoader debug mode with -Dorg.lwjgl.util.DebugLoader=true for better diagnostics.
16:02:26.295 | INFO  | Game output: Exception in thread "splashscreen-loop" java.lang.UnsatisfiedLinkError: Failed to locate library: liblwjgl.dylib
16:02:26.295 | INFO  | Game output: 	at org.lwjgl.system.Library.loadSystem(Library.java:164)
16:02:26.295 | INFO  | Game output: 	at org.lwjgl.system.Library.loadSystem(Library.java:63)
16:02:26.295 | INFO  | Game output: 	at org.lwjgl.system.Library.<clinit>(Library.java:51)
16:02:26.295 | INFO  | Game output: 	at org.lwjgl.glfw.GLFW.<clinit>(GLFW.java:30)
16:02:26.295 | INFO  | Game output: 	at org.terasology.engine.GLFWSplashScreen.run(GLFWSplashScreen.java:58)
16:02:26.295 | INFO  | Game output: 	at java.base/java.lang.Thread.run(Thread.java:829)
16:02:27.259 | INFO  | Game output: 16:02:27.258 [main] INFO  c.b.gdx.physics.bullet.NativeSupport - Loading JNBullet Library:bullet
16:02:27.260 | INFO  | Game output: java.lang.UnsatisfiedLinkError: Can't load library: /Users/skaldarnar/movingblocks/launcher/build/classes/games/OMEGA/NIGHTLY/5.3.0-rc.2/natives/macosx/libbullet-darwin-amd64.dylib
16:02:27.260 | INFO  | Game output: 	at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2633)
16:02:27.260 | INFO  | Game output: 	at java.base/java.lang.Runtime.load0(Runtime.java:768)
16:02:27.260 | INFO  | Game output: 	at java.base/java.lang.System.load(System.java:1837)
16:02:27.260 | INFO  | Game output: 	at com.badlogic.gdx.physics.bullet.NativeSupport.load(NativeSupport.java:117)
16:02:27.260 | INFO  | Game output: 	at com.badlogic.gdx.physics.bullet.Bullet.<clinit>(Bullet.java:28)
16:02:27.260 | INFO  | Game output: 	at org.terasology.engine.core.TerasologyEngine.<init>(TerasologyEngine.java:149)
16:02:27.260 | INFO  | Game output: 	at org.terasology.engine.core.TerasologyEngineBuilder.build(TerasologyEngineBuilder.java:31)
16:02:27.260 | INFO  | Game output: 	at org.terasology.engine.Terasology.call(Terasology.java:170)
16:02:27.260 | INFO  | Game output: 	at org.terasology.engine.Terasology.call(Terasology.java:70)
16:02:27.260 | INFO  | Game output: 	at picocli.CommandLine.executeUserObject(CommandLine.java:1933)
16:02:27.261 | INFO  | Game output: 	at picocli.CommandLine.access$1200(CommandLine.java:145)
16:02:27.261 | INFO  | Game output: 	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2332)
16:02:27.261 | INFO  | Game output: 	at picocli.CommandLine$RunLast.handle(CommandLine.java:2326)
16:02:27.261 | INFO  | Game output: 	at picocli.CommandLine$RunLast.handle(CommandLine.java:2291)
16:02:27.261 | INFO  | Game output: 	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2159)
16:02:27.261 | INFO  | Game output: 	at picocli.CommandLine.execute(CommandLine.java:2058)
16:02:27.261 | INFO  | Game output: 	at org.terasology.engine.Terasology.main(Terasology.java:139)
```

**Comment by soloturn** [comment]
should this be closed as duplicate of #5055 , as there, despite created after this one, there is more information, and a couple of fixes attached?
